keyboards/kumo/keymaps/LEdiodes/config.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "config_common.h"
+#define VENDOR_ID       0xFEAE
```

Comment:
Actually, most of this isn't needed. Only things changed from the base keyboard should be in here.